### ISSUE TITLE
Error when register name starts with "qreg" or "creg"

### ISSUE THREAD

**Issue by ChristopheVuillot** [QISKit/qiskit-sdk-py#82]
I found this bug : 
when creating a quantum register with create_quantum_register or create_classical_register, if one gives a name starting by "qreg" or "creg" then subsequent execution of a program using this register raises an error.
Here is an example code to reproduce the error : 
```python
from qiskit import QuantumProgram
import Qconfig

qprogram = QuantumProgram()
qprogram.set_api(Qconfig.APItoken, Qconfig.config["url"])

qtoto = qprogram.create_quantum_register("qregbliblablu",5)
ctiti = qprogram.create_classical_register("ctutu",5)
qcircuit= qprogram.create_circuit("mycircuit",[qtoto],[ctiti])

qcircuit.h(qtoto[0])
qcircuit.measure(qtoto[0],ctiti[0])

qprogram.execute(["mycircuit"], "ibmqx_qasm_simulator", shots=1000, max_credits=5, wait=10, timeout=240)
```

**Comment by diego-plan9** [comment]
Thanks for the report, @ChristopheVuillot! The parser seems indeed to lack robustness when handling tokens that start with reserved QASM words - we will look into this issue.

**Comment by diego-plan9** [comment]
Closed via #83 - as usual, thanks for reporting and don't hesitate on reopening on filing a new issue if needed!
